src/authz-module/libraries-manager/components/AddNewTeamMemberTrigger.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { FC, useState } from 'react';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { Button, Toast, useToggle } from '@openedx/paragon';
+import { Plus } from '@openedx/paragon/icons';
+
+import { useAddTeamMember } from '@src/authz-module/data/hooks';
+import AddNewTeamMemberModal from './AddNewTeamMemberModal';
+import messages from './messages';
+
+interface AddNewTeamMemberTriggerProps {
+  libraryId: string;
+}
+
+const DEFAULT_FORM_VALUES = {
+  users: '',
+  role: '',
+};
+
+const AddNewTeamMemberTrigger: FC<AddNewTeamMemberTriggerProps> = ({
+  libraryId,
+}) => {
+  const intl = useIntl();
+  const [isOpen, open, close] = useToggle(false);
+  const [additionMessage, setAdditionMessage] = useState<string | null>(null);
+  const [formValues, setFormValues] = useState(DEFAULT_FORM_VALUES);
+
+  const { mutate: addTeamMember, isPending: isAddingNewTeamMember } = useAddTeamMember();
+
+  const handleChangeForm = (e: React.ChangeEvent<HTMLTextAreaElement | HTMLSelectElement>) => {
+    const { name, value } = e.target;
+    setFormValues((prev) => ({
+      ...prev,
+      [name]: value,
+    }));
+  };
+
+  const handleAddTeamMember = () => {
```

Comment:
There is only one thing that is missing here, that is after a partial success the input, should display the error users in red (o just display the error users for now) and remove the completed ones.

Initial input
<img width="403" height="272" alt="image" src="https://github.com/user-attachments/assets/1ee066cf-35ff-4355-8997-1122009c2863" />
After submitting
<img width="575" height="408" alt="image" src="https://github.com/user-attachments/assets/ea3d37c0-0615-41dd-8714-2411b51eaa2a" />

Reply:
Solved :+1: